README.md

```diff
@@ -114,6 +114,30 @@ optional arguments:
                         effectively excluded via the --include_paths option.
 ```
 
+## Running with Docker
+
+First, enter the directory containing the git repository
+
+```
+cd /path/to/git
+```
+
+On Linux or MacOS, run the following:
+
+```
+docker run --rm -v `pwd`:/proj dxa4481/trufflehog file:///proj
+```
+
+On Windows, using powershell, run the following instead:
+
+```
+docker run --rm -v $(pwd):/proj dxa4481/trufflehog file:///proj
```

Comment:
This is valid on Linux and macOS, too. No need to provide different commands.

Reply:
Sorry about the delay. I've fused the command line